doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst

```diff
@@ -0,0 +1,1302 @@
+.. _ncs_release_notes_250:
+
+|NCS| v2.5.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+The following list includes the summary of the most relevant changes introduced in this release.
+
+* Added :ref:`support <software_maturity>` for the following features:
+
+  * Bluetooth Low Energy:
+
+    * LE Power Control request procedure that was introduced in |NCS| 2.2.0 as experimental is now supported.
+
+  * Bluetooth Mesh:
+
+    * Networked Lighting Controls (NLC), a set of Bluetooth Mesh profiles for standardized wireless lighting control.
+    * Bluetooth Mesh 1.1 features introduced in nRF Connect SDK 2.4.0 as experimental are now supported.
+
+      .. note::
+         Samples built with Bluetooth Mesh 1.1 will be labeled as experimental when built.
+         The experimental flag in the Zephyr code base could not be removed in time for the release unfortunately, but this is planned to be changed.
+
+  * Sidewalk:
+
+    * :ref:`ug_sidewalk` is now integrated as part of |NCS|, supporting nRF52840 and nRF5340 SoCs.
+
+  * Wi-Fi:
+
+    * Enhanced Wi-Fi scan API, providing optimizations and high user configurability for the Wi-Fi scanning functionality on nRF70 Series.
+    * Network agnostic connectivity manager, allowing seamless use of IP-based protocols regardless of underlying transport (Cellular IoT, Wi-Fi).
+    * :ref:`aws_iot`, :ref:`azure_iot_hub`, :ref:`download_sample`, :ref:`https_client`, :ref:`udp_sample`, and :ref:`wifi_zephyr_samples` are now supported on nRF7002 DK.
+    * New samples: :ref:`wifi_twt_sample`, `wifi_shutdown_sample`, and :ref:`wifi_wfa_qt_app_sample`
```

Comment:
missing :ref: for the  `wifi_shutdown_sample`

Reply:
Thanks Ulf! Fixed.